planetiler-core/src/main/java/com/onthegomap/planetiler/geo/TileCoord.java

```diff
@@ -101,7 +108,7 @@
     return "{x=" + x + " y=" + y + " z=" + z + '}';
   }
 
-  public double progressOnLevel(TileExtents extents) {
+  public double progressOnLevel(TileOrder order, TileExtents extents) {
```

Comment:
For hilbert I think we want this to be an approximation like `encoded - startForZoom(z)` with a TODO about how we could make it exact later if desired.